scripts/evaluation/olsconfig.yaml

```diff
@@ -0,0 +1,56 @@
+# olsconfig.yaml sample for local ollama server
+#
+# 1. install local ollama server from https://ollama.com/
+# 2. install llama3.1:latest model with:
+#       ollama pull llama3.1:latest
+# 3. Copy this file to the project root of cloned lightspeed-service repo
+# 4. Install dependencies with:
+#       make install-deps
+# 5. Start lightspeed-service with:
+#       OPENAI_API_KEY=IGNORED make run
+# 6. Open https://localhost:8080/ui in your web browser
+#
+llm_providers:
+  - name: ollama
+    type: openai
+    url: "http://localhost:11434/v1/"
+    models:
+      - name: "mistral"
+      - name: 'llama3.2:latest'
+  - name: my_rhoai
+    type: openai
+    url: "https://granite3-8b-wisdom-model-staging.apps.stage2-west.v2dz.p1.openshiftapps.com/v1"
+    credentials_path: ols_api_key.txt
+    models:
+      - name: granite3-8b
+ols_config:
+  # max_workers: 1
+  reference_content:
+    # product_docs_index_path: "./vector_db/vector_db/aap_product_docs/2.5"
```

Comment:
any reason why these lines are commented out?

Reply:
so, `road-core` eval framework has multiple `eval_modes` and these input would be used with `ols_rag` mode when RAG pipeline needs to be evaluated which involves use of `vector_db` path. But for `vanilla` mode input to `vector_db` isn't required, so have commented the section as this is the example `olsconfig.yaml`. If needed I can un-comment.